Sources/TensorFlow/Layers/Dropout.swift

```diff
@@ -73,3 +73,31 @@ public struct Dropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
     }
   }
 }
+
+public struct GaussianDropout<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
+  @noDerivative
+  public let probability: Double
+  
+  @noDerivative
+  public let standardDeviation: Tensor<Scalar>
+  
+  public init(probability: Double, standardDeviation: Scalar) {
+    precondition(
+      0...1 ~= probability,
+      "Probability must be a value between 0 and 1 (inclusive) but is \(probability)")
+    self.probability = probability
+    self.standardDeviation = Tensor<Scalar>(standardDeviation)
+  }
+  
+  @differentiable
+  public func callAsFunction(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
+    switch Context.local.learningPhase {
+    case .training:
+      let noise = Tensor<Scalar>(randomNormal: input.shape, mean: Tensor<Scalar>(0),
+                                 standardDeviation: self.standardDeviation)
+      return input + noise
```

Comment:
If this is gaussian dropout this should be `input * noise`? See [here](https://github.com/tensorflow/tensorflow/blob/e5bf8de410005de06a7ff5393fafdf832ef1d4ad/tensorflow/python/keras/layers/noise.py#L120)

Reply:
@vballoli It looks like you implemented the `GaussianNoise` layer, You should either rename this to `GaussianNoise` or implement the `GausianDropout` Layer.

Reply:
Yeah @Shashi456 my bad. I've implemented noise here, I've already completed GaussianDropout too, just need some time on pushing it and re-submitting the PR. Sorry about the delay